py3status/modules/diskdata.py

```diff
@@ -89,7 +89,9 @@ def space_and_io(self):
 
         self.values = {'disk': self.disk if self.disk else 'all'}
 
-        if '{read}' in self.format or '{write}' in self.format or '{total}' in self.format:
+        if (self.py3.format_contains(self.format, 'read') or
+                self.py3.format_contains(self.format, 'write') or
+                self.py3.format_contains(self.format, 'total')):
```

Comment:
maybe format_contains should learn about lists.  Anyhow this is good for now, thanks